dotcom-rendering/src/web/components/Links.importable.tsx

```diff
@@ -251,18 +244,18 @@ export const Links = ({
 			{/* Since 'subscriptions' is only rendered on the client, rendering it within
 				a div is required to prevent DOM elements getting mis-matched during hydration  */}
 			<div css={linkWrapperStyles}>
-				{showSupporterCTA && supporterCTA !== '' && (
+				{supporterCTA !== '' && (
```

Comment:
We don't need the conditional `supporterCTA !== ''` check here, as we're not using `supporterCTA` in the link whilst the SFD test is running.